adapters/taboola/taboola.go

```diff
@@ -3,45 +3,59 @@ package taboola
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/prebid/openrtb/v17/native1"
+	nativeResponse "github.com/prebid/openrtb/v17/native1/response"
 	"github.com/prebid/openrtb/v17/openrtb2"
 	"github.com/prebid/prebid-server/adapters"
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
 	"github.com/prebid/prebid-server/openrtb_ext"
 	"net/http"
+	"text/template"
 )
 
 type adapter struct {
-	endpoint string
+	endpoint *template.Template
 }
 
+const (
+	NATIVE_ENDPOINT_PREFIX  = "native"
+	DISPLAY_ENDPOINT_PREFIX = "display"
+)
```

Comment:
scope of `NATIVE_ENDPOINT_PREFIX` and `DISPLAY_ENDPOINT_PREFIX` is limited to taboola package. so we can make them private

```
const (
   nativeEndpointPrefix = "native"
   displayEndpointPrefix = "display"
)
```

Reply:
@onkarvhanumante  added them to an internal scope